src/main/java/org/broadinstitute/hellbender/tools/funcotator/filtrationRules/AlleleFrequencyUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.tools.funcotator.filtrationRules;
+
+import org.broadinstitute.hellbender.tools.funcotator.FilterFuncotations.AlleleFrequencyDataSource;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public abstract class AlleleFrequencyUtils {
+
+    /**
+     * Build a {@link FuncotationFiltrationRule} matching Funcotations from variants with a
+     * maximum MAF less than some threshold.
+     *
+     * @param maxMaf the MAF threshold to check in the rule. Must be in the range [0, 1]
+     * @param afDataSource the allele frequency data source (ExAC or gnomAD) with which the original VCF was Funcotated.
+     * @return a {@link FuncotationFiltrationRule} matching Funcotations with a MAF (AC/AN)
```

Comment:
I think it's safe to get rid of `(AC/AN)` in the comment at this level since it's only calculated this way for ExAC